Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -455,19 +459,24 @@ def PyExec(self):  # noqa
                                    OutputWorkspace=sam_ws_name,
                                    Tolerance=self.COMPRESS_TOL_TOF)  # 5ns/
 
-            # make sure there are no negative values - gsas hates them
-            if self.getProperty("PushDataPositive").value != "None":
-                addMin = self.getProperty("PushDataPositive").value == "AddMinimum"
-                api.ResetNegatives(InputWorkspace=sam_ws_name,
-                                   OutputWorkspace=sam_ws_name,
-                                   AddMinimum=addMin,
-                                   ResetValue=0.)
-
             # write out the files
             # FIXME - need documentation for mpiRank
             if mpiRank == 0:
                 if self._scaleFactor != 1.:
                     api.Scale(sam_ws_name, Factor=self._scaleFactor, OutputWorkspace=sam_ws_name)
+                if self._offsetFactor != 0.:
+                    api.ConvertToMatrixWorkspace(InputWorkspace=sam_ws_name,
```

Comment:
Is there any reason for this not to use the `simpleapi` apart from historical?

Reply:
I see it imports `simpleapi` as `api`. There is nothing wrong with it except causing a bit of a confusion.